src/_includes/install/php-versions-2.3.md

```diff
@@ -1,6 +1,4 @@
 Magento supports PHP 7.3.
 
 {:.bs-callout-warning}
-PHP 7.1 has reached [End of Life](https://www.php.net/supported-versions.php). To maintain PCI compliance, Magento should not be run on unsupported software.
-Installing from GitHub will no longer work with Magento 2.3.4/PHP 7.1.
-The only way to install 2.3.4 with PHP 7.1.x is with Composer.
\ No newline at end of file
+PHP 7.1 and 7.2 have reached [End of Life](https://www.php.net/supported-versions.php). To maintain PCI compliance, Magento should not be run on unsupported software. Magento 2.3.x supports PHP 7.3 only and has not been tested with PHP 7.1 or 7.2.
```

Comment:
Historically speaking this is not really true because Magento 2.3.0, 2.3.1 and 2.3.2 never supported PHP 7.3

Quoting from https://devdocs.magento.com/guides/v2.3/release-notes/release-notes-2-3-3-open-source.html:

> Magento 2.3.3 now supports PHP 7.3.x (tested with PHP 7.3.8) and PHP 7.2.x (tested with 7.2.21).

But whatever ... 🙂